lms/static/js/courseware/link_clicked_events.js

```diff
@@ -0,0 +1,16 @@
+;(function(define) {
```

Comment:
It is as far as I can tell from looking at the codebase, though it's not a convention I've encountered previously.